src/Repository/UserRepository.php

```diff
@@ -49,32 +102,14 @@ public function loadUserByIdentifier(string $identifier): ?User
             ->getOneOrNullResult();
     }
 
-    // /**
-    //  * @return User[] Returns an array of User objects
-    //  */
-    /*
-    public function findByExampleField($value)
+    public function getById(Ulid $ulid): Querybuilder
```

Comment:
Does this not duplicate doctrine's own `find()` https://www.doctrine-project.org/projects/doctrine-orm/en/2.16/reference/working-with-objects.html#by-primary-key

Reply:
I think we need this because of 
->setParameter('user', $ulid, 'ulid');
'ulid' as type